README.md

```diff
@@ -64,7 +61,7 @@ Or you can pass them in on the command line:
 
 ### Using custom binaries
 
-The `autospotting.zip` file can be generated by building it locally. Further instructions are available in the [AutoSpotting repo](https://github.com/autospotting/autospotting).
+The `autospotting.zip` file can be generated by building it locally. Further instructions are available in the [AutoSpotting repo](https://github.com/AutoSpotting/AutoSpotting/blob/master/CUSTOM_BUILDS.md).
 
 You can also download the latest official nightly build available [here](https://cloudprowess.s3.amazonaws.com/nightly/autospotting.zip).
```

Comment:
This gives a 404, the correct URL is https://cloudprowess.s3.amazonaws.com/nightly/lambda.zip

Please update the name to `lambda.zip` throughout this documentation.

Reply:
Sure, I have done that now.